playwright/tests/homePageElements.spec.ts

```diff
@@ -0,0 +1,101 @@
+import { Given, Then } from "@cucumber/cucumber";
+import { expect } from "@playwright/test";
+import { CustomWorld } from "../support/setup";
+
+Given("I am on the DRB home page", async function (this: CustomWorld) {
+  return await this.page.goto("https://digital-research-books-beta.nypl.org/");
+});
+
+Then("I see the site name", async function (this: CustomWorld) {
+  await expect(
+    this.page.locator("//h1[contains(text(), 'Digital Research Books')]")
+  ).toBeVisible();
+});
+
+Then("I see the Home breadcrumb", async function (this: CustomWorld) {
+  await expect(
+    this.page.locator(
+      "//span[@class='breadcrumb-label'][contains(text(), 'Home')]"
+    )
+  ).toBeVisible();
+});
+
+Then("I see the Research breadcrumb", async function (this: CustomWorld) {
+  await expect(
+    this.page.locator(
+      "//span[@class='breadcrumb-label'][contains(text(), 'Home')]"
+    )
+  ).toBeVisible();
+});
+
+Then(
+  "I see the Digital Research Books Beta breadcrumb",
+  async function (this: CustomWorld) {
+    await expect(
+      this.page.locator(
+        "//span[@class='breadcrumb-label'][contains(text(), 'Digital Research Books Beta')]"
+      )
+    ).toBeVisible();
+  }
+);
+
+Then("I see the site name H1", async function (this: CustomWorld) {
+  await expect(this.page.locator("//h1")).toContainText(
+    "Digital Research Books"
+  );
+});
+
+Then("I see the intro text", async function (this: CustomWorld) {
+  await expect(
+    this.page.locator("//div[@data-testid='hero-content']/p/span")
+  ).toContainText(
+    "Find millions of digital books for research from multiple sources"
+  );
+});
```

Comment:
We don't necessarily care that this specific div > p > span contains this text. What we care about is if that text is visible to users. For this reason I would change this test to:

```
Then("I see the intro text", async function (this: CustomWorld) {
  await expect(
    this.page.locator(
      "text=Find millions of digital books for research from multiple sources"
    )
  ).toBeVisible();
});
```

Playwright seems to be able to locate text on the page using this `text=` selector, which I think we should start using instead of lots of nested xpath selectors. Here's the reference guide: https://playwright.dev/docs/selectors#text-selector